CMakeLists.txt

```diff
@@ -328,10 +328,8 @@ SET(abs_top_builddir "${CMAKE_CURRENT_BINARY_DIR}")
 SET(abs_top_srcdir "${CMAKE_CURRENT_SOURCE_DIR}")
 
 SET(CC_VERSION "${CMAKE_C_COMPILER}")
-
-# Build *FLAGS for libpio.settings.
-SET(CFLAGS "${CMAKE_C_FLAGS} ${CMAKE_C_FLAGS_${CMAKE_BUILD_TYPE}}")
-SET(CPPFLAGS "${CMAKE_CPP_FLAGS} ${CMAKE_CPP_FLAGS_${CMAKE_BUILD_TYPE}}")
+SET(FC_VERSION "${CMAKE_Fortran_COMPILER}")
```

Comment:
Is FC_VERSION set in the autotools build?

Reply:
@jedwards4b did you check how this works in the autotools build? I'm not sure it will know what to do with FC_VERSION...